doc/autotest/property/properties/Elastic-make.md

```diff
@@ -0,0 +1,43 @@
+## Elastic make
+
+**Step 1.** The `DeformedStructureSet` module in [pymatgen.analysis.elasticity.strain](https://pymatgen.org/pymatgen.analysis.elasticity.strain.html) is used to generate a set of independently deformed structures. `equi.stress.out` file is written to record the equilibrium stress in the Elastic directory. For the example in the previous section, `equi.stress.out` should be in `confs/mp-*/elastic_00`.
+
+**Step 2.** If there are `init_from_suffix` and `output_suffix` parameter in the `properties` part, the [refine process](../../refine/Refine-get-started-and-input-examples.html) follows. Else, the deformed structure (`POSCAR`) and strain information (`strain.out`) are written in the task directory, for example, in `confs/mp-*/elastic_00/task.000000`.
```

Comment:
This link does not work.

![image](https://user-images.githubusercontent.com/9496702/187098569-a0e440fc-2699-4e93-ba77-50581685dd1b.png)

Reply:
Please also check other links.